x11-misc/revelation/revelation-0.5.3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_8 )
+
+inherit python-single-r1 autotools gnome2
+
+DESCRIPTION="A password manager for GNOME"
+HOMEPAGE="https://revelation.olasagasti.info/ https://github.com/mikelolasagasti/revelation"
+SRC_URI="https://github.com/mikelolasagasti/revelation/releases/download/${P}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RESTRICT="test"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/pycryptodomex[${PYTHON_USEDEP}]
+                dev-python/pygobject[${PYTHON_USEDEP}]
+                dev-libs/libpwquality[python,${PYTHON_USEDEP}]
+	')
+        x11-libs/gtk+:3
+"
```

Comment:
Note that watch carefully the upstream dependency requirements, something is unlisted here. And you can use 
```
|| ( x1 x2 )
```
syntax to choose between *pycryptodome* and *pycryptodomex*.

Reply:
https://github.com/mikelolasagasti/revelation/blob/2a0b42276377fa7c44ac2e510f86ac33d36aa6b6/meson.build#L17 Here's the dep list.